lib/lte_link_control/lte_lc.c

```diff
@@ -886,6 +886,39 @@ int lte_lc_edrx_req(bool enable)
 	return err;
 }
 
+int lte_lc_pdp_context_set(enum lte_pdp_type type, const char *apn,
+			bool ip4_addr_alloc, bool nslpi, bool securePCO)
+{
+	static const char * const pdp_type_lut[] = {
+		"IP", "IPV6", "IPV4V6"
```

Comment:
This relies on the order in the enum staying as is, so it should be documented in the header file that the order is important, or alternative logic should be implemented here.